database/migrations/2020_07_18_122946_update_users_table_add_twitter_handle_field.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class UpdateUsersTableAddTwitterHandleField extends Migration
+{
+    public function up()
+    {
+        Schema::table('users', function (Blueprint $table) {
+            $table->string('twitter_handle')->after('email')->nullable();
```

Comment:
Let's keep this simple as `twitter`. I want to change `github_username` to just `github` as well in the future in a different PR. 

Also: merge this migration into the other one to keep them as low as possible.